app/src/js/components/Executions/execution-status-graph.js

```diff
@@ -27,16 +27,21 @@ class ExecutionStatusGraph extends React.Component {
 
     const workflow = JSON.parse(stateMachine.definition);
 
+    console.log('WORKFLOW: ' + JSON.stringify(workflow));
+
     var events = getExecutionEvents(executionHistory);
+
     var graph = workflowToGraph(workflow);
+    console.log('GRAPH: ' + JSON.stringify(graph));
     addEventsToGraph(events, graph);
     this.g = draw(graph);
     var render = new dagre.render();
     var svg = d3.select('svg');
```

Comment:
Fwiw, when I was working on my ticket, I noticed that since this is selecting 'svg', it grabs the first one it finds. When the datepicker is present, the svg within the start date is what is selected and causes the weird styling. If we add a class to the svg we're rendering here and then select for that, it should solve the issue. If that's the reason we're hiding the datepicker header, than we can avoid that by doing this 

```
var svg = d3.select('.visual');
.
.
.
.
render () {
  return (
    <svg className='visual'></svg>
  );
}
```